Source/SplashScreen/Mixins/Main.lua

```diff
@@ -0,0 +1,157 @@
+-- TODO All this needs Localization
+-- We should probably move this into another file too
+local MESSAGES = {
+  {
+    Version = "8.3.0",
```

Comment:
To localise this we need to replace every string with an AUCTIONATOR_L_SPLASH_830_DESCRIPTION, AUCTIONATOR_L_SPLASH_830_SECTION_BUGS, AUCTIONATOR_L_SPLASH_830_BUGS_LINE_1, etc